docs/learning/ui/appui-react/ToolSettings.md

```diff
@@ -75,6 +75,56 @@ export class SampleToolWithSetting extends PrimitiveTool {
 }
 ```
 
+### Example using latest ToolSettings helper methods to simplify setting up properties for Tool Settings UI
```

Comment:
Do we need to call out latest here?

Reply:
Removed the word 'latest' - it made since to me at the time - but it would not make sense to anyone in a few months.